core/tests/puppeteer-acceptance-tests/specs/logged-in-learner/goes-through-the-sign-in-flow.spec.ts

```diff
@@ -19,30 +19,79 @@
  * LI.SI. Learner goes through the sign-in flow
  */
 
+import testConstants from '../../utilities/common/test-constants';
 import {UserFactory} from '../../utilities/common/user-factory';
 import {LoggedInUser} from '../../utilities/user/logged-in-user';
 import {LoggedOutUser} from '../../utilities/user/logged-out-user';
+import {ReleaseCoordinator} from '../../utilities/user/release-coordinator';
+
+const ROLES = testConstants.Roles;
 
 describe('Logged In Learner', function () {
   const loggedInUser: LoggedInUser & LoggedOutUser = Object.assign(
     new LoggedInUser(),
     new LoggedOutUser()
   );
+  let releaseCoordinator: ReleaseCoordinator;
+
+  beforeAll(async function () {
+    // Create release coordinator to enable redesigned learner dashboard.
+    releaseCoordinator = await UserFactory.createNewUser(
+      'releaseCoordinator',
+      'release_coordinator@example.com',
+      [ROLES.RELEASE_COORDINATOR]
+    );
+    await releaseCoordinator.enableFeatureFlag(
+      'show_redesigned_learner_dashboard'
+    );
+  }, 600000);
 
-  it('should be able to log in', async function () {
-    // Navigate to sign in page.
+  it('should be able to login and see Learner Dashboard', async function () {
+    // Click on "Sign In" button and fill email.
     await loggedInUser.openBrowser();
     await loggedInUser.navigateToSignUpPage();
 
-    // Sign Up.
+    // Enter email to proceed to username page.
     await loggedInUser.enterEmail('logged_in_user@example.com');
+
+    // Verify the signup page is shown with username field after entering email.
+    await loggedInUser.waitForUsernameInputToBeVisible();
```

Comment:
You can add it to `enterEmail` function as a post-check. No need to call it separately.